### DISCUSSION TITLE
Adding auto start on boot

### DISCUSSION THREAD

**Discussion by DAH69** [Mygod/VPNHotspot#470] [Ideas]
Hi 
Is it possible to add a feature to auto start the mobile hotspot and the wlan on system boot?

And if it's possible
Please 🙏
Do it

Thank y'all.

**Comment by Mygod** [comment]
Not planned. You can use a third party app that turns on the hotspot on boot, and use the monitor feature (which can be auto started).

**Reply by DAH69** [reply]
How did I not think of it, thank you
